frontend/frontend/src/components/Game.vue

```diff
@@ -1,12 +1,10 @@
 <template>
 	<div class="board">
 		<div class="board-wrapper">
+			<h2> Matched: {{this.countMatched}}</h2>
+			<button v-on:click="routeToProfile">Stop Game</button>
 			<Board :cardData="shuffled" @updateActive="updateActive" />
 		</div>
-		<div>
-			<div class="timer">timer</div>
-			<button @click="returnHome">Leave Game</button>
```

Comment:
I think the leave game button is still relevant, as it takes them to the home page.